src/block_cache/block_chunkcache.c

```diff
@@ -499,6 +499,10 @@ __wt_chunkcache_get(WT_SESSION_IMPL *session, WT_BLOCK *block, uint32_t objectid
     if (!F_ISSET(chunkcache, WT_CHUNKCACHE_CONFIGURED))
         return (ENOTSUP);
 
+    /* Only cache read-only tiered objects. */
+    if (!block->readonly)
+        return (__wt_read(session, block->fh, offset, size, dst));
```

Comment:
Should we simply return successfully from this function? The caller should handle the remaining steps since chunkcache_hit will be false.

Reply:
Good spot - done in 8002baa.